scripts/switch_gui.py

```diff
@@ -1,11 +1,11 @@
 import argparse
 import logging
 import sys
-from os import path
 
 from PyQt5.QtWidgets import QApplication
 
 import switchtool.ui as switch_ui
+from switchtool.switch.switch import switch_types
```

Comment:
I think it's worth renaming this / capitalizing it to be clear that it's a constant, rather than a function that changes the type of an object.

It's actually a mapping from switch type to Surveryer class, so maybe `SWITCH_NAME_TO_SURVEYER`?